content/200-concepts/100-components/01-prisma-schema/06-relations/500-troubleshooting-relations.mdx

```diff
@@ -228,6 +228,7 @@ This however tells Prisma to expect **two** separate one-to-many relationships.
 
 The following example is the correct way to define an implicit many-to-many relationship.
 
+TODO Does the below work? Or do we need to name both sides of the relation?
```

Comment:
Question for @prisma/schema-team

Reply:
Yeah, that won't work. There are multiple issues in that schema:
- there are duplicate relation fields on both models, (categories twice on Post and posts twice on Category)
- if you name a relation on one side, you also need to name it on the other one (you can have one unnamed relation fields pair though)

This would work: 
model Post {
   id         Int        @id @default(autoincrement())
  title      String
  categories Category[] @relation("Category")
  categories2 Category[]
}
model Category {
  id    Int    @id @default(autoincrement())
  name  String
  posts Post[] @relation("Category")
  posts2 Post[] 
}

Reply:
It looks like the example has delete/add highlighting though? So there isn't a duplication of fields, it's removing one line and adding the other

Reply:
Removing TODO comment so I can take this PR out of draft. I'll keep this conversation unresolved for now though and update example if necessary.